<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radioButtonTitle">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>44</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>제목</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonActor">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>44</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>배우</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDirector">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>44</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>감독</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonGenre">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>44</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>장르</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonYear">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>44</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>년도</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonCountry">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>44</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>국가</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>15</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>11</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>250</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sortNameButton">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>73</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>이름순 정렬</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortYearButton">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>99</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>년도순 정렬</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortRateButton">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>127</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>평점순 정렬</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tempButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>화면전환(임시)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
